.changeset/curly-foxes-brake.md

```diff
@@ -0,0 +1,14 @@
+---
+'@backstage/plugin-scaffolder': patch
+---
+
+Apply `defaultValue` props in `MultiEntityPicker`
+
+```diff
```

Comment:
You can remove the diff bit from the changeset, don't need this. Just the text above describing the change is great :tada:

Reply:
diff bit removed~ thank you!